addons/web/static/src/core/notifications/notification.xml

```diff
@@ -2,17 +2,14 @@
 <templates xml:space="preserve">
 
     <t t-name="web.NotificationWowl" owl="1">
-        <div t-attf-class="o_notification {{className}}"
-        role="alert" aria-live="assertive" aria-atomic="true">
-            <div class="o_notification_body">
-                <button type="button" class="close o_notification_close"
-                aria-label="Close" t-on-click="props.close">
-                    <span class="d-inline" aria-hidden="true">×</span>
-                </button>
-                <div t-if="props.title" class="font-weight-bold o_notification_title mb-1" t-esc="props.title"/>
-                <div t-if="props.message" class="mr-auto o_notification_content">
-                    <t t-out="props.message"/>
-                </div>
+        <div t-attf-class="o_notification border border-{{props.type}} bg-white mb-2 position-relative"
```

Comment:
Missing `props.className` to handle transition:
```suggestion
        <div t-attf-class="o_notification {{props.className}} border border-{{props.type}} bg-white mb-2 position-relative"
```

Reply:
^^ I've just committed the same change locally, thanks anyway 🙌